### ISSUE TITLE
Unable to use with react 15

### ISSUE THREAD

**Issue by shishircc** [rrag/react-stockcharts#92]
How can I use this with react 15 ? 

Using version 0.4 does not work because it requires react 0.14 and version next is not installable using npm.

**Comment by rrag** [comment]
not without some hacks.

`0.5` is not ready yet, and I cannot promise any timeline.

**Hack to make 0.4 work with react 15:**

your `npm install` will report an error but will still complete.
after you `npm install` 

```
cd node_modules/react-stockcharts/node_modules
rm -rf react react-dom
ln -s ../../react react
ln -s ../../react-dom react-dom
```

after this there will still be a problem where the mouse crosshair pointer and dragging pointer when you pan will not be visible, it will always be a default pointer.

to fix that add

```
	.react-stockcharts-grabbing-cursor {
		cursor: grabbing;
		cursor: -moz-grabbing;
		cursor: -webkit-grabbing;
	}
	.react-stockcharts-crosshair-cursor {
		cursor: crosshair;
	}
	.react-stockcharts-toottip-hover {
		pointer-events: all;
		cursor: pointer;
	}
```
to your .css file

This is all a hack since 0.5 is not out yet. It goes without saying these should be removed once 0.5 is available

**Comment by rrag** [comment]
fixed, published in 0.5
